api/models.py

```diff
@@ -45,6 +46,14 @@ class LessonStudent(models.Model):
   created_at = models.DateTimeField(auto_now_add=True)
   updated_at = models.DateTimeField(auto_now=True)
 
+  def student_average_score(student_id):
+    score = LessonStudent.objects.filter(student_id=student_id).aggregate(Avg('score'))
+    return score
+
+  def lesson_average_score(lesson_id):
+    score = LessonStudent.objects.filter(lesson_id=lesson_id).aggregate(Avg('score'))
+    return score
+
```

Comment:
Great job working with aggregates and getting used to this ORM. looks really clean

Reply:
Glad to know models function similar to the models in rails. This looks great